newdle/templates/result_email.txt

```diff
@@ -1,7 +1,14 @@
-Hi,
+{{ title }}
 
-{{ creator }} picked a final date for the event "{{ title }}".
+Hi {{ participant_name }}!
+
+''{{ title }}'' has a final date.
+
+{{ creator }} has chosen {{ date }} from {{ start_time }} to {{ end_time }} ({{ timezone }}) for ''{{ title }}''.
```

Comment:
tbh i'm not sure if this is more readable than the previous version... you now have a super long line in a plaintext email...

I would focus on making the HTML mails prettier in this PR (wasn't the issue about those anyway?)

Reply:
This is the HTML version
![Newdle_result_mail_notification v2](https://user-images.githubusercontent.com/71437950/95090889-15c02e00-0726-11eb-8fa4-efff5388e037.png)


I really don't know for what is the .txt file, so I basically try to put the same format as the HTML. I recovered again the old version and I put some more information

Reply:
Not everyone wants to read HTML emails, so we include both a HTML and plain text version - many mail clients then let you choose which version you prefer to see.